doc/debugging.rst

```diff
@@ -1,45 +1,62 @@
+#########
 Debugging
-=========
+#########
 
-Debugging in Spyder is supported thanks to the following Python modules:
-
-* `pdb`: the Python debugger, which is included in Python standard library.
-
-* `winpdb`: a graphical frontend to `pdb`, which is an external package
-  (in the :doc:`editor`, press F7 to run `winpdb` on the currently edited
-  script).
+**Debugging** in Spyder is supported through integration with the enhanced ``ipdb`` debugger in the :doc:`ipythonconsole`.
+This allows breakpoints and the execution flow to be viewed and controlled right from the Spyder GUI, as well as with all the familiar IPython console commands.
 
 .. image:: images/debugging/debugging_console.png
    :align: center
    :alt: A Spyder IPython console window, showing the ipdb debugger in action
 
-Debugging with pdb
-------------------
+|
+
+
+===================
+Debugging with ipdb
+===================
+
+Spyder offers the following debugging features integrated into the native GUI:
+
+.. image:: images/menu/menu_debug.png
+   :align: right
+   :width: 50%
+   :alt: Spyder's debug menu, showing options to set, show and clear breakpoints, and control debugger execution flow
 
-The Python debugger is partly integrated in Spyder:
+* Multiple means of setting and clearing normal and conditional breakpoints for any line in a file opened in the :doc:`editor`.
 
-* Breakpoints may be defined in the :doc:`editor`.
+  * By selecting the respective option from the Debug menu.
+  * Through pressing a configurable keyboard shortcut (:kbd:`F12` for normal, or :kbd:`Shift-F12` for conditional breakpoints by default).
+  * By double-clicking to the left of the line number in an open file.
+  * With an ``ipdb.set_trace()`` statement in your code.
+  * Interactively, using the ``b`` magic command in an ``ipdb`` session.
```

Comment:
Remove `magic` here. `b` is a regular Pdb command.

Reply:
Thanks, done.